README.md

```diff
@@ -40,494 +40,196 @@ Ensure packages are installed with correct version numbers by running:
-`{ startDate: momentPropTypes.momentObj, endDate: momentPropTypes.momentObj }`
-as an argument.
-```js
-  onDatesChange: PropTypes.func,
-```
+We provide a handful of components for your use. Each of these have a handful of props required for a fully functional interactive date picker as well as a large number of props for customizing the look and feel of the inputs, calendar, and
+even individual days. You can see what each of the props do in the [live demo](http://airbnb.io/react-dates/) or explore
+how to properly wrap the pickers in the [examples folder](https://github.com/airbnb/react-dates/tree/master/examples).
 
-**Focus:**
+#### DateRangePicker
+The `DateRangePicker` is a fully controlled component that allows users to select a date range. You can control the selected
+dates using the `startDate`, `endDate`, and `onDatesChange` props as shown below. The `DateRangePicker` also manages internal
+state for partial dates entered by typing (although `onDatesChange` will not trigger until a date has been entered
+completely in that case). Similarly, you can control which input is focused as well as calendar visibility (the calendar is
+only visible if `focusedInput` is defined) with the `focusedInput` and `onFocusChange` props as shown below.
 
-The `focusedInput` prop indicates which of the two inputs is currently focused, if either. You can import the `START_DATE` and `END_DATE` constants from `react-dates/constants`.
-```js
-  focusedInput: PropTypes.oneOf([START_DATE, END_DATE]),
-```
-
-`onFocusChange` is the callback necessary to update the focus state being held in the parent component and pass that back down to the `DateRangePicker` as a prop. `onFocusChange` receives either `START_DATE`, `END_DATE`, or `null` as an argument.
-```js
-  onFocusChange: PropTypes.func,
-```
-
-**Date selection rules:**
-
-The value of `minimumNights` indicates the minimum number of days between the start date and the end date.
-```js
-  minimumNights: PropTypes.number,
-```
-
-To indicate which days are blocked from selection, you may provide a function to the `isDayBlocked` prop. As of v1.0.0, we allow blocked dates inside of ranges.
-```js
-  isDayBlocked: PropTypes.func,
+Here is the minimum *REQUIRED* setup you need to get the `DateRangePicker` working:
+```jsx
+<DateRangePicker
+  startDate={this.state.startDate} // momentPropTypes.momentObj or null,
+  endDate={this.state.endDate} // momentPropTypes.momentObj or null,
+  onDatesChange={({ startDate, endDate }) => this.setState({ startDate, endDate })} // PropTypes.func.isRequired,
+  focusedInput={this.state.focusedInput} // PropTypes.oneOf([START_DATE, END_DATE]) or null,
+  onFocusChange={focusedInput => this.setState({ focusedInput })} // PropTypes.func.isRequired,
+/>
 ```
 
-`isOutsideRange` indicates which days are out of selectable range.
```

Comment:
Just to notice, I think better to not cut existing docs unless storybook lacks in some parts. for instance in storybook not clear what this function should return